devices/stello.js

```diff
@@ -0,0 +1,59 @@
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const stello = {
+    fz: {
+          power: {
+            cluster: 'hvacThermostat',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty('16392')) {
+                    return {power: msg.data["16392"]};
+                }
+				else {
+					return 0;
+				}					
+            },
+		  },		  
+		  energy: {
+            cluster: 'hvacThermostat',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty('16393')) {
+                    return {energy: parseFloat(msg.data["16393"]) / 1000};
+                }
+            },
+        },
+    },
+};
+
+const definition = {
+	zigbeeModel: ['HT402'], 
+	model: 'HT402', 
+	vendor: 'Stello', 
+	description: 'Hilo Stelpro thermostat', 
+	fromZigbee: [fz.legacy.stelpro_thermostat, fz.legacy.hvac_user_interface, stello.fz.power, stello.fz.energy],  

```

Comment:
Since you are using the stelpro converter, is this realy a stelpro device? If yes move this to: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/stelpro.js

Reply:
It is sold and rebranded by "Hilo" and it returns "vendor: Stello" and model "HT402" https://www.hiloenergie.com/en-ca/products/energie/thermostat-intelligent-hilo-thermostats_1/
 (but the looks and features seem to match the Stelpro Allia thermostat SAT402ZB https://www.stelpro.com/fr/produits/thermostat-maison-intelligente-allia-sat402zb/).

Reply:
I propose to move it to `stelpro.js` then (look like a whitelabel), update vendor to `Stelpro` and description to `Hilo thermostat`.

Reply:
Ok I can do this. Sorry for the newbie questions, but can I recopy the stelpro.js file locally and modify it on my machine to try the changes and make sure I don't break anything and it still works before submitting the changes and the PR?

Reply:
You can just move it, I will do a final check to ensure it won't break anything :)

Reply:
Ok but I also want to check my device will still work also like that

Reply:
@Koenkk  I tried migrating to Stelpro.js..  let me know if it's ok... The tests still fail...  I'm also thinking that probably Stelpro thermostat support getting power and energy the same way but it might have been never implemented..? It would be a pretty cool upgrade to give them.    Also if it does not what about the documentation is there a way to tell that this thermostat supports power and energy but not the others? Thank you!